### ISSUE TITLE
Error handling?

### ISSUE THREAD

**Issue by rtnielson** [sebi/googl-php#5]
https://developers.google.com/url-shortener/v1/getting_started#errors

How does this (sebi) code catch error handling? I don't see anything offhand.

**Comment by sebi** [comment]
Feel free to add error handling, I will happily merge it. ;-)

**Comment by mes01** [comment]
Notice: Undefined property: stdClass::$id in /var/data/www/Googl.class.php on line 72

**Comment by jason-engage** [comment]
Yeah this bug blocks the script. Any way to quickly fix this? Probably very simple fix.
